great-idea/index.html

```diff
@@ -13,11 +13,65 @@
     <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
   <![endif]-->
 </head>
-
-  <body>
-
-    <!-- Copy and paste your HTML from the first UI project here -->
-      
-    
+<body>
+  <div class="container page-index">
+    <header class="main-navigation">
+      <a class="logo" href="index.html"><img src="img/logo.png" alt="Great Idea! Company logo."></a>
+      <nav>
+        <a href="services.html">Services</a>
+        <a href="#">Product</a>
+        <a href="#">Vision</a>
+        <a href="#">Features</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+      </nav>
+    </header>
+    <main>
+      <section class="splash">
+        <div class="call-to-action">
+          <h1><span>Innovation</span> <span>On</span> <span>Demand</span></h1>
+          <a href="#" class="button">Get Started</a>
+        </div>
+        <img src="img/header-img.png" alt="Image of a code snippet.">
+      </section>
+      <section class="content">
+        <div class="blurb-row">
+          <div class="blurb">
+            <h2>Features</h2>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+          </div>
+          <div class="blurb">
+            <h2>About</h2>
+            <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+          </div>
+        </div>
+        <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+        <div class="blurb-row">
```

Comment:
Might not want to name all these class the same. For example you need flexbox(display:flex) on this on to style its  3 div child siblings. & you might not need flexbox on line 38 but since they are the same class name the styles will apply to both. When creating HTML, only set class names when needed.